src/main/java/com/exasol/adapter/dialects/postgresql/PostgresSQLSqlGenerationVisitor.java

```diff
@@ -283,8 +284,9 @@ public String visit(final SqlFunctionAggregateGroupConcat function) throws Adapt
             builder.append(") ");
             return builder.toString();
         } else {
-            throw new SqlGenerationVisitorException(
-                    "List of arguments of SqlFunctionAggregateGroupConcat should have one argument.");
+            throw new SqlGenerationVisitorException(ExaError.messageBuilder("E-PGVS-8")
```

Comment:
I would remove the check for arguments number here because I prefer to check it on the parsing level

Reply:
If you don't wan't to remove it, then maybe it's also fatal with mitigation as the previous one

Reply:
Removed the validation and creating a ticket for doing it in vs-common-java